examples/simulation/Tutorial2/macros/compare_seed_value.C

```diff
@@ -17,4 +33,8 @@ int compare_seed_value(TString filename, UInt_t initial_seed) {
     std::cout << "Seed value from file : " << seed << std::endl;
     return 1;
   }
+}
+
+int main() {
+  return compare_seed_value("",1);
```

Comment:
Why are you adding this here, if you remove it in a later commit again?

Reply:
Because it was added by accident. This was only meant for testing and should not end up in the PR.

Reply:
Can you please remove it from the relevant commit and possibly squash some commits together?

Reply:
Done.